DBM-Raids-WoTLK/RubySanctum/Halion.lua

```diff
@@ -161,7 +176,9 @@ function mod:SPELL_CAST_SUCCESS(args)--We use spell cast success for debuff time
 end
 
 function mod:SPELL_AURA_APPLIED(args)--We don't use spell cast success for actual debuff on >player< warnings since it has a chance to be resisted.
-	if args.spellId == 74792 then
+	local spellId = args.spellId
+	local destcId = args:GetDestCreatureID()

```

Comment:
This can be moved to Corporeality, no need to grab creatureid for other events

Reply:
destcid I agree, but the spellid is checked on every if case, thus accessing the arg table on every if

Reply:
Yeah I didn't even mean to highlight spellId, that's fine